README.md

```diff
@@ -34,10 +34,11 @@ Before running any of the examples, the following steps must be performed.
         git clone https://github.com/apache/accumulo-examples.git
         mvn clean package
 
-4. Specify Accumulo connection information.  All examples read connection information from a 
-   properties file. Copy the template and edit it.
+4. Specify Accumulo connection information in `conf/accumulo-client.properties`.  Some old examples
+   still read connection information from an examples.conf file so that should also be configured.
```

Comment:
I take it you are just going through and updating each example one by one?  I think that is fine, but maybe make a note somewhere (create an issue?) to consolidate the two files when its done.